jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -177,6 +178,16 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Store the provided SSH key in state.
+
+      @method _setSSHKey
+      @param {String} key The SSH key.
+    */
+    _setSSHKey: function(key) {
+      this.setState({sshKey: key});
```

Comment:
It does not seem that we can pass more than one key when creating the model.
LP or github API is a good idea, but it does not affect this branch. Here we are only setting up a system to store the key, where does that key come from is another question. Initially we'll just support copy/pasting a key. Later, we can improve that as you suggested.